### DISCUSSION TITLE
Not possible to click on resource annotations with caret browsing?

### DISCUSSION THREAD

**Discussion by rhoadsstevens** [ManifoldScholar/manifold#3482] [Accessibility]
STEPS TO REPRODUCE:
1. Use Chrome browser. Press F7 to turn on caret browsing.
2. With caret browsing, and using keyboard commands and not a mouse, navigate to a resource annotation. (So one of those little cubes that indicate an annotation with a resource.)
3. Observe that the caret cursor skips over the resource cube. 

OBSERVED BEHAVIOR:
When using caret browsing, I find that I can't select a resource annotation and click on it. 

EXPECTED BEHAVIOR: 
As I'm navigating a Manifold text with caret browsing and encounter a resource annotation, I should be able to click on it. Then, once I click on it, I should be able to get into the Manifold media player and navigate it.

**Comment by dananjohnson** [comment]
We've redesigned resource annotations for the next version of Manifold (v10). As part of this, we've ensured that the toggles and the modal dialogs they control are fully keyboard-accessible.
